fuzz/brontide/fuzz_utils.go

```diff
@@ -48,11 +48,16 @@ var (
 			return nil, err
 		}
 
-		priv, _ := secp256k1.PrivKeyFromBytes(eBytes)
+		priv, _ := privKeyFromBytes(eBytes)
 		return priv, nil
 	})
 )
 
+func privKeyFromBytes(eBytes []byte) (*secp256k1.PrivateKey, *secp256k1.PublicKey) {
```

Comment:
Wouldn't it be better to just avoid doing this?  I looks like every case except one just discards the public key anyway and thus it'll be faster to avoid computing it.

Reply:
Nice catch. Thanks for the review!